src/index.html

```diff
@@ -1,11 +1,13 @@
-<html>
-	<head>
-		<meta charset="utf-8">
-	  	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	  	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
-		<title>Cesium + Webpack</title>
-	</head>
-	<body>
-		<div id="cesiumContainer"></div>
-	</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <!-- Include the CesiumJS JavaScript and CSS files -->
+  <script src="https://cesium.com/downloads/cesiumjs/releases/1.84/Build/Cesium/Cesium.js"></script>
+  <link href="https://cesium.com/downloads/cesiumjs/releases/1.84/Build/Cesium/Widgets/widgets.css" rel="stylesheet">
```

Comment:
CDN is ok for most web applications, but painfull for intranet environment with no internet connections...

> I'm not quite sure whether cesium should be put into "dependencies" or "devDependencies"

Reply:
The purpose of this guide is to provide a general framework for users to get up and running with CesiumJS and Webpack. In the future, we can create a guide that caters better to applications with no internet access.